draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -3110,8 +3111,8 @@ respond with an INIT ACK chunk using the same parameters it sent in its
 original INIT chunk (including its Initiate Tag, unchanged), provided
 that no NEW address has been added to the forming association.
 If the INIT chunk indicates that a new address has been added to the
-association, then the entire INIT chunk MUST be discarded, and SHOULD NOT do any
-changes to the existing association.
+association, then the entire INIT chunk MUST be discarded, and SHOULD NOT
+change the state of any existing association.
```

Comment:
We are processing the INIT chunk in the context of one specific association. So I think referring to `any existing association` might not be a precise as possible. To improve the wording, I changed the text to:
```
If the INIT chunk indicates that a new address has been added to the
association, then the entire INIT chunk MUST be discarded, and the state of
the existing association SHOULD NOT be changed.
```

Reply:
The next revision will contain the text I suggested. If you don't agree, please unresolve the conversation.